topo_processor/metadata/metadata_loaders/metadata_loader_imagery_historic.py

```diff
@@ -260,5 +250,17 @@ def is_valid_centroid(self, item: Item, centroid: Dict[str, Any]) -> bool:
         return True
 
     def add_linz_geospatial_type(self, item: Item, photo_type: str) -> None:
-
         item.linz_geospatial_type = historical_imagery_photo_type_to_linz_geospatial_type(photo_type)
+
+    def add_stac_extensions(self, item: Item) -> None:
+        item.add_extension(StacExtensions.eo.value)
```

Comment:
Im not sure im sold on these extensions all being grouped down here, to me it makes much more sense to add the extension when their corresponding keys are added

Are any of these extensions only added when the metadata for them is present? 

for instance `eo:bands` should only be added if there is a asset and it has bands? granted all (?) of the historical imagery assets should have a band count?

Reply:
I believe extensions should be added anyway. I can move them back into the separate functions.